src/Qir/Execution/Tools/Microsoft.Quantum.Qir.Tools.csproj

```diff
@@ -53,45 +53,4 @@
     </Compile>
   </ItemGroup>
 
-  <ItemGroup>
```

Comment:
This logic was important for the qir-tools to work correctly. Did it get moved somewhere?